src/main/java/com/iciql/TableInspector.java

```diff
@@ -115,6 +115,7 @@ void read(DatabaseMetaData metaData) throws SQLException {
             while (rs.next()) {
                 IndexInspector info = new IndexInspector(rs);
                 if (info.type.equals(IndexType.UNIQUE)) {
+                    if(info.name == null) continue;
```

Comment:
`info.name` is used below the if in line 126ff.
I would rather even put it one line up - it only happend to me for `IndexType.UNIQUE` but could happen for other types.